lang/fr.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: fr\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Usage: vrms-rpm [options]"
+msgid "msg_help_option_ascii"    
+msgstr "    Afficher l'art ASCII rms quand aucun progiciels non-gratuits sont trouvés,\n"
+       "    ou quand des progiciels non-gratuits comprennent 100%% ou plus du total."
+msgid "msg_help_option_explain"
+msgstr "    Afficher les licenses pour justifier la classification de gratuité pendant\n"
+       "    le listage des progiciels."
+msgid "msg_help_option_help"
+msgstr "    Afficher ce menu d'aide puis quitter"
+msgid "msg_help_option_list"
+msgstr "    En surcroît d'afficher un sommaire des progiciels gratuits et non-gratuits,"
+       "    afficher leurs noms. La valeur par défaut est 'nonfree'."
+msgid "msg_help_option_version"
+msgstr "    Afficher la version puis quitter."
+msgid "msg_list_noargument"
+msgstr "%s: --list option requiert un argument."
+msgid "msg_list_badargument"
+msgstr "%s: l'argument pour --list option doit être 'none', 'free', 'nonfree' ou 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: option inconnue '%s'"
+msgid "msg_total_free"
+msgstr "%d progiciels gratuits"
```

Comment:
With "Free Software", the "free" part is not about price (as in "free beer"), but about freedom. Taking a look at the [French Wikipedia](https://fr.wikipedia.org/wiki/Logiciel_libre), it uses "libre" and "propriétaire" for "free" and "proprietary".

Reply:
Absolutely! I knew the way I wrote it sounded weird, but I didn't know a better way to phrase it. Thanks!

Reply:
You seem to have missed this single occurence of "gratuits".  :)